src/components/MainContact/index.jsx

```diff
@@ -1,36 +1,27 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import { useInView } from 'react-intersection-observer';
 import { useSpring, animated } from 'react-spring';
 import cn from 'classnames';
-import { charityAPI } from '../../clients';
+import { useCharityAPI } from '../../clients';
 import Loader from './ContentLoader';
 import './styles.scss';
-const MainContact = () => {
-  const [data, setData] = useState({});
-  const [loading, setLoading] = useState(true);
-  const [error, setError] = useState(false);
-  const [errorMessage, setErrorMessage] = useState('');
 
-  const getData = () => {
-    setLoading(true);
-    charityAPI('/main-contacts')
-      .then(({ data }) => {
-        setData(data);
-        setLoading(false);
-        setError(false);
-      })
+const MainContactContainer = () => {
+  const { data, loading, dataError } = useCharityAPI('/main-contacts');
+  return (
+    <MainContact
+      data={data}
+      loading={loading}
+      error={dataError}
+      errorMessage={dataError.message}
```

Comment:
Do not forget to remove `errorMessage` as you can get message using the previous one